packages/nimble-components/src/utilities/models/validator.ts

```diff
@@ -0,0 +1,21 @@
+import type { ValidityObject } from '../../table/types';
+import { Tracker } from './tracker';
+
+/**
+ * Generic Validator Utility extends Tracker Utility for validation purposes
+ */
+export class Validator<
+    ValidationFlags extends readonly string[]
+> extends Tracker<ValidationFlags> {
+    public constructor(validationFlags: ValidationFlags) {
+        super(validationFlags);
+    }
```

Comment:
Same as the other, constructor isn't doing anything anymore, think you can leave it off and calls will go right to the parent